docs/reference/gpu-arch/gpu-arch-spec-overview.rst

```diff
@@ -215,27 +647,54 @@ For a more detailed explanation refer to the :ref:`specific documents and guides
+  Amount of memory available on the GPU. Some SKUs might have a different amount of VRAM.
 Compute Units
-  Number of compute units on the GPU.
-Wavefront Size
-  Amount of work-items that execute in parallel on a single compute unit. This is equivalent to the warp size in HIP.
+  Number of compute units (CUs) on the GPU.
+Warp Size
+  Amount of work-items that execute in a lock-step fashion.
+VGPR File
+  Size of the Vector General Purpose Register (VGPR) file. Holds data used in vector instructions.
+  GPUs with matrix cores also have AccVGPRs, which are Accumulation General Purpose Vector Registers, specifically used in matrix instructions.
+SGPR File
+  Size of the Scalar General Purpose Register (SGPR) file. Holds data used in scalar instructions.
 LDS
-  The Local Data Share (LDS) is a low-latency, high-bandwidth scratch pad memory. It is local to the compute units, shared by all work-items in a work group. In HIP this is the shared memory, which is shared by all threads in a block.
+  The Local Data Share (LDS) is a low-latency, high-bandwidth scratch pad memory. It is local to the WGP on RDNA GPUs and shared by the CUs, on CDNA and GCN GPUs it is local to the CUs. The LDS is shared by all work-items in a work group. In HIP this is the shared memory, which is shared by all threads in a block.
+
+Instinct Terminology
+""""""""""""""""""""
+
+The following section explains the terminology used in CDNA and GCN architectures.
+
+GCD
+  Graphics Compute Die.
 L3 Cache
   Size of the level 3 cache. Shared by all compute units on the same GPU. Caches vector and scalar data and instructions.
 L2 Cache
-  Size of the level 3 cache. Shared by all compute units on the same GCD. Caches vector and scalar data and instructions.
+  Size of the level 2 cache. Shared by all compute units on the same GCD. Caches vector and scalar data and instructions.
 L1 Vector Cache
   Size of the level 1 vector data cache. Local to a compute unit. Caches vector data.
 L1 Scalar Cache
   Size of the level 1 scalar data cache. Usually shared by several compute units. Caches scalar data.
 L1 Instruction Cache
   Size of the level 1 instruction cache. Usually shared by several compute units.
-VGPR File
-  Size of the Vector General Purpose Register (VGPR) file. Holds data used in vector instructions.
-  GPUs with matrix cores also have AccVGPRs, which are Accumulation General Purpose Vector Registers, specifically used in matrix instructions.
-SGPR File
-  Size of the Scalar General Purpose Register (SGPR) file. Holds data used in scalar instructions.
-GCD
-  Graphics Compute Die.
+
+Radeon Terminology
+""""""""""""""""""
+
+The following section explains the terminology used in RDNA architectures.
+
+WGP
+  Work group processor. Consists of two compute units.
+Infinity Cache
+  Size of the infinity cache. This is a memory attached last level cache, functioning as a L3 cache. It is shared by all compute units. Caches vector and scalar data and instructions.
```

Comment:
```suggestion
  Size of the infinity cache. This is a memory attached last level cache, functioning as an L3 cache. It is shared by all compute units. Caches vector and scalar data, and instructions.
```

a**n** L3 cache.

> functioning as an L3 cache

What's the **function** of an L3 cache compared to an L2 cache for example? I would rather say that it is also _called_ an L3 cache in documentation.

The last sentence is hard to read, I think adding a comma helps convey the intended meaning: data (both vector and scalar) and instructions.

Reply:
Removed the glossary, as dicsussed, and added an additional (not yet working) reference to the HIP programming guide